site/layouts/partials/scripts.html

```diff
@@ -26,36 +26,45 @@
   document.querySelectorAll('.btn-edit').forEach(btn => {
     btn.addEventListener('click', event => {
       const htmlSnippet = event.target.closest('.bd-code-snippet').querySelector('.bd-example').innerHTML
-      StackBlitzSDK.openBootstrapSnippet(htmlSnippet)
+
+      // Get extra classes for this example except '.bd-example'
+      const classes = event.target.closest('.bd-code-snippet').querySelector('.bd-example').classList
+      if (classes) {
+        classes.remove('bd-example')
+      }
+
+      const jsSnippet = event.target.closest('.bd-code-snippet').querySelector('.btn-edit').getAttribute('data-js-snippet')
+      StackBlitzSDK.openBootstrapSnippet(htmlSnippet, jsSnippet, classes)
     })
   })
 
-  StackBlitzSDK.openBootstrapSnippet = snippet => {
+  StackBlitzSDK.openBootstrapSnippet = (htmlSnippet, jsSnippet, classes) => {
     const markup = `<!doctype html>
 <html lang="en">
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link href="{{ .Site.Params.cdn.css }}" rel="stylesheet">
     <title>Bootstrap Example</title>
+    <${'script'} src="{{ .Site.Params.cdn.js_bundle }}"></${'script'}>
   </head>
-  <body>
+  <body class="${classes}">
```

Comment:
This is needed by [Checks & Radios - Indeterminate](https://deploy-preview-36352--twbs-bootstrap.netlify.app/docs/5.2/forms/checks-radios/#indeterminate), [Toasts - Translucent](https://deploy-preview-36352--twbs-bootstrap.netlify.app/docs/5.2/components/toasts/#translucent), etc.

Reply:
its ok by me. Either way. I would like a `padding/margin` class (ex:`m-3`)  to give some space around example

Reply:
Yes let's do this :ok_hand:  Added a `.p-3` via https://github.com/twbs/bootstrap/pull/36352/commits/cf2e9287a2c6e900504cf3ecba6220a3da478d1d. I'll see in the next PRs more focused on the examples that don't work because of some missing CSS if we need to make it optional or not.